sugoi-api-rest-services/src/test/java/fr/insee/sugoi/services/controller/OrganizationControllerTest.java

```diff
@@ -260,7 +261,7 @@ public void getNextLocationInSearchResponse() {
       assertThat(
           "Location header gives next page",
           mockMvc.perform(requestBuilder).andReturn().getResponse().getHeader("Location"),
-          is("http://localhost/realms/domaine1/organizations?size=2&offset=2"));
+          is("http://localhost/realms/domaine1/organizations?size=2&searchToken"));
```

Comment:
why not mySearchToken here ? Why does the test pass ?

Reply:
because i forgot to change one things in the controller part i changed it now